File: corona-project-backend/corona_backend/Models/GraphsModel.cs
namespace corona_backend.Models
{
    public class GraphsModel
    {
        public List<SevenDaysCasesAverageModel> SevenDaysCasesAverage { get; set; }

        public List<DailyCasesModel> DailyCases {  get; set; }
    }
}

File: corona-project-frontend/src/models/graphsModel.ts
import { dailyCasesModel } from "./dailyCasesModel";
import { sevenDaysCasesAverageModel } from "./sevenDaysCasesAverageModel";

export interface graphsModel{
    sevenDaysCasesAverage: sevenDaysCasesAverageModel[],
    dailyCases: dailyCasesModel[]
}